docs/CHANGELOG.md

```diff
@@ -74,12 +74,14 @@ extra bounds from `AnimatablePrefab` and `AnimationSetPrefab` ([#1435])
 * Removed `NetEvent::Custom` and added `NetEvent::Packet(NetPacket)` ([#1523])
 * Fixed update is no longer frame rate dependent ([#1516])
 * Display the syntax error when failing to parse sprite sheets  ([#1526])
-
+* `NetConnection` is automatically created when client starts sends data to server. ([#1539])
+* User will receive `NetEvent::Connected` on new connection and `NetEvent::Dissconnected` on disconnect. ([#1539])
 
 ### Removed
+- Removed all `NetEvent's` because they were not used. ([#1539])
+- Removed filter logic, because it didn't do anything, will be added back in a later version (NetFilter, FilterConnected). ([#1539])
```

Comment:
Both were meant to be defined/used by the user.
Filter was implemented before... not sure where the handling code went.

Reply:
Alright, but those events weren't interpreted anywhere in the code, so what's the point of having them? 

You are right about the filtering, I am not sure when it was removed, and in fact, I tried to look it up but could not find it, but since the connection events weren't interpreted the filtering was basicly doing nothing. There was also a comment to point that out somewhere. Filtering will come back but can be done at a later time.